### PR TITLE
app-text/master-pdf-editor: new ebuild

### PR DESCRIPTION
**Pull request by jpataias**
Add latest version of Master PDF Editor.

> Master PDF Editor is the optimal solution for editing PDF files in Linux. It enables you to create, edit, view, encrypt, sign and print interactive PDF documents.

In collaboration with @jfmcarreira.

### CODE REVIEW

**Comment by monsieurp** [comment]
Thanks for the PR. @ulm what's your take on the licence of this software?

**Comment by ulm** [comment]
> Thanks for the PR. @ulm what's your take on the licence of this software?

AFAICS, no problem adding this to the tree. However, the license
doesn't grant the right to redistribute, so the ebuild will need
mirror restriction.

(Also, about the license file itself: Please rewrap these long lines
to a reasonable length, like 70 chars. At the end of the file a
newline is missing.)

**Comment by jpataias** [comment]
Added the required changes do ebuild and license file.

File: app-text/master-pdf-editor/master-pdf-editor-4.3.61.ebuild [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+	exeinto ${dest}
+	doexe ${bin_name}
+
+	dosym ${dest}/${bin_name} /opt/bin/${bin_name}
+	make_desktop_entry ${bin_name} "Master PDF Editor ${PV}" ${dest}/${bin_name}.png "Office;Graphics;Viewer" "MimeType=application/pdf;application/x-bzpdf;application/x-gzpdf;\nTerminal=false"
+}
+
+pkg_postinst() {
+	sed -e '/^Exec=/s/masterpdfeditor4/masterpdfeditor4 %f/' -i /usr/share/applications/masterpdfeditor4-${PN}.desktop
```
Comment (monsieurp) [inline app-text/master-pdf-editor/master-pdf-editor-4.3.61.ebuild]: Add a `|| die` to the end of the line here please.

File: app-text/master-pdf-editor/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>joaompssantos@gmail.com</email>
```
Comment (monsieurp) [inline app-text/master-pdf-editor/metadata.xml]: You will maintain this package. Please file a bug about this package to make sure your email in recorded in Bugzilla. Also, add the proxy-maintainer project. Developers of this project will proxy your changes.

Comment (jpataias) [reply app-text/master-pdf-editor/metadata.xml]: Should I go to bugs.gentoo.org and a new report under Gentoo Linux --> New packages?

File: app-text/master-pdf-editor/master-pdf-editor-4.3.61.ebuild [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+EAPI=6
+
+inherit xdg-utils eutils versionator
+
+DESCRIPTION="Master PDF Editor is the complete solution for viewing and editing PDF files."
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+
+SRC_URI="http://get.code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+RESTRICT="mirror"
```
Comment (ulm) [inline app-text/master-pdf-editor/master-pdf-editor-4.3.61.ebuild]: Small nitpick: The usual convention is for RESTRICT to go to the end of the second variables block (i.e., LICENSE, SLOT, KEYWORDS, [IUSE], [RESTRICT]).

File: app-text/master-pdf-editor/metadata.xml (hunk 2 above)
Comment (monsieurp) [reply app-text/master-pdf-editor/metadata.xml]: Yes.

File: app-text/master-pdf-editor/master-pdf-editor-4.3.61.ebuild (hunk 3 above)
Comment (jpataias) [reply app-text/master-pdf-editor/master-pdf-editor-4.3.61.ebuild]: Will do, thanks!

**Comment by jpataias** [comment]
Corrected required changes and added a bug report: https://bugs.gentoo.org/635690.

Apparently, this was already done before for a different version: https://bugs.gentoo.org/610948. However I can't find the package on Gentoo tree or on the overlays.

**Pull request closed without merging**